workitem/workitem_revision.go

```diff
@@ -0,0 +1,44 @@
+package workitem
+
+import (
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+)
+
+const (
+	// CREATE_WORKITEM_REVISION_TYPE a work item creation
+	CREATE_WORKITEM_REVISION_TYPE = 1
+	// DELETE_WORKITEM_REVISION_TYPE a work item deletion
+	DELETE_WORKITEM_REVISION_TYPE = 2
+	// UPDATE a work item update
+	UPDATE_WORKITEM_REVISION_TYPE = 4
```

Comment:
thanks for the link. After so many years of coding in Java, I still have some habits that don't fit in Golang